tests/console/snapper_cleanup.pm

```diff
@@ -48,7 +48,12 @@ sub snapper_cleanup {
     for (1 .. $scratch_size_gb) { assert_script_run("$snap_create", 500); }
     assert_script_run('sync');
     script_run "echo There are `$snaps_numb` snapshots BEFORE cleanup";
-    assert_script_run("snapper cleanup number", 300);    # cleanup created snapshots
+    # cleanup created snapshots
+    # softfail the test if hitting bsc#1206814, so that the subsequent test runs are not affected
+    if ((script_run('snapper cleanup number', timeout => 300) != 0) && is_leap) {
```

Comment:
This has two issues:

1. Any issue on leap would now get marked as `bsc#1206814`
2. Any other issue on any other system would be ignored.

As a suggestion: On Leap - Check if we can use `script_output` to check, if the command was successful. If unsuccessful, check if `Failure (error.something).` is therein and then record the softfailure.  
On all other systems - use `assert_script_run`.

Reply:
You are correct! 
I will use script_output